src/Orchard.Web/Modules/Orchard.Roles/Conditions/RoleCondition.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Orchard.Conditions.Services;
+using Orchard.ContentManagement;
+using Orchard.Roles.Models;
+using Orchard.Security;
+using System;
+using System.Linq;
+
+namespace Orchard.Roles.Conditions {
+    public class RoleRuleProvider : IConditionProvider {
```

Comment:
Because `IConditionProvider` is an event handler you should not take a reference on this, just copy the interface in your project, and remove the reference. The implementation will be called because their interface names match. MAGIC !